### DISCUSSION TITLE
Questions regarding the competition rules

### DISCUSSION THREAD

**Discussion by qiaoling123455** [duchallenge-team/dude#4] [Q&A] [answered]
1. Does the Task 1 have two evaluation phases and each of them use different test set data? 
2. The final submission date for phase 1 is March 15th and April 1st for phase 2, is this correct?
3. When will the final testing data be available for submission for both phase 1 and phase 2?
Thank you very much for your time and consideration.

**Comment by Jordy-VL** [comment, marked as answer]
1. yes, there will be (implicitly) two test sets for phase 1 and phase 2. However, this will be provided all together as one, otherwise one would know which questions come from which set, defeating the purpose of out-of-domain detection.
2,3. see the updated schedule on the RRC platform

>> repeating the announcement here

Dear participants, 

Following inquiries regarding the submission schedule, we have opted to better align the submissions for evaluation phase 1 (test set for Task 1) and for evaluation phase 2 (domain shift and selective question-answering).

We hope that this renewed schedule gives participants more clarity and time to work on their submissions (3 weeks, 1 week for method description) for either evaluation phase. We would urge participants to submit method descriptions (just a couple of paragraphs to understand the overall idea) together with their submissions as the competition report deadline is strict. We would not want to disqualify any team for not providing their method description in time.  

When giving participants access to questions from the test set, we will provide a mixture of the questions from evaluation phase 1 and phase 2.
Submissions for phase 1 would need to only provide the answers key and preferably answers_confidence for calibration evaluation.
Submissions for phase 2 would need to as well provide both the keys answers_confidence and answers_abstain for all question-answer pairs. 

Cheers, 

The DUDEs
